.husky/_/husky.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 if [ -z "$husky_skip_init" ]; then
   debug () {
-    [ "$HUSKY_DEBUG" = "1" ] && echo "husky (debug) - $1"
+    if [ "$HUSKY_DEBUG" = "1" ]; then
```

Comment:
what's this change for?

Reply:
its probably my idea optimizing the code.  this doesn't have a negative effect so I can leave it alone or revert it.